Dockerfile

```diff
@@ -13,7 +13,7 @@ RUN groupadd conda; \
   usermod -g conda root
 
 # Ownership & permissions based on https://docs.anaconda.com/anaconda/install/multi-user/#multi-user-anaconda-installation-on-linux
-COPY --from=condaforge/mambaforge:22.9.0-2 --chown=root:conda --chmod=770 /opt/conda /opt/conda
+COPY --from=condaforge/mambaforge:23.1.0-1 --chown=root:conda --chmod=770 /opt/conda /opt/conda
```

Comment:
Is your fork up-to-date?

The `main` branch currently shows that this version is being used :thinking:: https://github.com/rapidsai/mambaforge-cuda/blob/main/Dockerfile#L16


It was updated in #29.

Reply:
Looks like it's a few commits out of date.

I think that's also why the PR workflows aren't running.

Reply:
Sorry... I rebased on `upstream/main`.